runtime-watcher/internal/watchermetrics/metrics.go

```diff
@@ -0,0 +1,70 @@
+package watchermetrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type WatcherMetrics struct {
+	requestDurationGauge               prometheus.Gauge
+	failedKCPRequestsTotalCounter      prometheus.Counter
+	kcpRequestsTotalCounter            prometheus.Counter
+	admissionRequestsErrorTotalCounter prometheus.Counter
+	admissionRequestsTotalCounter      prometheus.Counter
+}
+
+const (
+	requestDuration             = "watcher_request_duration"
+	failedKCPRequestsTotal      = "watcher_failed_kcp_total"
+	kcpRequestsTotal            = "watcher_kcp_requests_total"
+	admissionRequestsErrorTotal = "watcher_admission_request_error_total"
+	admissionRequestsTotal      = "watcher_admission_request_total"
+)
+
+func NewMetrics() *WatcherMetrics {
```

Comment:
A general question, how do we know which skr cluster these metrics coming from? We don't need additional label like kyma name to indicate it?

Reply:
As far as I know yes and the documentation (internal) also states: 

> [...] and also applies the extra labels to all metrics specific to the runtimes from where the metrics are coming in.